css/login.css

```diff
@@ -0,0 +1,164 @@
+body{
+	background-color: #555555;
+	margin: 0;
+	font-family: 'Roboto', sans-serif;
+	min-width: 1200px;
+}
+
+.rectangle{
+	width: 100%;
+	height: 640px;
+}
+
+.firstRectangle{
+	background-image: url('../img/feed.jpg');
+	background-blend-mode: overlay;
+}
+
+.whiteRectangle{
+	width: inherit;
+	height: inherit;
+	background-color: #CBF6FF;
+	opacity: 0.3;
+	position: absolute;
+}
+
+.logInBox{
+	width: 375px;
+	height: 400px;
+	margin-left: 400px;
+	background-color: #F5F5F5;
+	opacity: 1!important;
+	position: absolute;
+	margin-top: 140px;
+	box-shadow: 0px 0px 8px black;
+	border-radius: 3px;
+}
+
+.headLogIn{
+	width: inherit;
+	background-color: #1ab2ff;
+	height: 75px;
+	box-shadow: 0px 3px 10px grey;
+	display: table;
+}
+
+.headLogIn > div{
+	color: white;
+	height: 75px;
+	font-size: 26px;
+	text-align: center;
+	line-height: 75px;
+}
+
+.contentLogIn{
+	width: 90%;
+	height: 70%;
+	margin: 0 auto;
+	margin-top: 20px;
+}
+
+.logFormClass{
+	margin-top: 20px;
+	position: absolute;
+	width: inherit;
+	background-color: #F5F5F5;
+	text-align: center;
+}
+
+.logFormInput{
+	border: 0;
+	width: 100%;
+	height: 50px;
+	margin-bottom: 10px;
+	background-color: inherit;
+	padding-left: 10px;
+	font-size: 15px;
+	border-bottom: 1px solid grey;
+}
+
+.logFormButton{
+	margin: 0 auto;
+	width: 120px;
+	height: 30px;
+	background-color: #1ab2ff;
+	color: white;
+	margin-top: 50px;
+	border: 0;
+	box-shadow: 0px 3px 15px grey;
+}
+
+/* Second Rectangle */
+
+.secondRectangle{
```

Comment:
Well here depends. If you know that your page will be extended or not. But, of course there is always a better name that can be found for classes.